derive_builder_core/src/builder_field.rs

```diff
@@ -75,6 +75,14 @@ impl<'a> ToTokens for BuilderField<'a> {
     }
 }
 
+impl<'a> BuilderField<'a> {
+    /// Emits a struct field initializer that initializes the field to `Default::default`.
+    pub fn default_initializer_tokens(&self) -> TokenStream {
+        let ident = self.field_ident;
+        quote!{ #ident : ::derive_builder::export::core::default::Default::default(), }
```

Comment:
Is there a reason to use `Default::default()` here instead of `::derive_builder::export::core::option::Option::None`?

Reply:
I don't remember. Maybe it was to work with optional fields that aren't wrapped in `None`.

Reply:
Ah, I think it's for the `PhantomData` case you mentioned.